msteams-platform/toolkit/Microsoft-365-Agents-Toolkit-CLI.md

```diff
@@ -0,0 +1,517 @@
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| [`atk doctor`](#atk-doctor)             | Prerequisite checker for building Microsoft 365 Apps.                                                              |
+| [`atk new`](#atk-new)                   | Create a new Microsoft 365 App.                                                                                    |
+| [`atk add`](#atk-add)                   | Add feature to your Microsoft 365 App.                                                                             |
+| [`atk auth`](#atk-auth)                 | Manage Microsoft 365 and Azure accounts.                                                                           |
+| [`atk entra-app`](#atk-install)         | Manage the Microsoft Entra app in the current application.                                                         |
+| [`atk env`](#atk-env)                   | Manage environments.                                                                                               |
+| [`atk help`](#atk-help)                 | Show Microsoft 365 Agents Toolkit CLI help.                                                                        |
+| [`atk install`](#atk-install)           | Sideload a given application package across Microsoft 365.                                                         |
+| [`atk launchinfo`](#atk-launchinfo)     | Get launch information of an acquired Microsoft 365 App.                                                           |
+| [`atk list`](#atk-list)                 | List available Microsoft 365 App templates and samples.                                                            |
+| [`atk provision`](#atk-provision)       | Run the provision stage in `m365agents.yml` or `m365agents.local.yml`.                                             |
+| [`atk deploy`](#atk-deploy)             | Run the deploy stage in `m365agents.yml` or `m365agents.local.yml`.                                                |
+| [`atk package`](#atk-package)           | Build your Microsoft 365 App into a package for publishing.                                                        |
+| [`atk validate`](#atk-validate)         | Validate the Microsoft 365 App using manifest schema, validation rules, or test cases.                             |
+| [`atk publish`](#atk-publish)           | Run the publish stage in `m365agents.yml`.                                                                         |
+| [`atk preview`](#atk-preview)           | Preview the current application.                                                                                   |
+| [`atk update`](#atk-update)             | Update the Microsoft 365 App manifest to Teams Developer Portal.                                                   |
```

Comment:
Update the Microsoft 365 App manifest to **Teams** Developer Portal. => Update the Microsoft 365 App manifest to Developer Portal.